src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2293,12 +2293,12 @@
       text: az staticwebapp list
 """
 
-helps['staticwebapp browse'] = """
+helps['staticwebapp show'] = """
     type: command
     short-summary: Show details of a static app.
     examples:
     - name: Show static app in a subscription.
-      text: az staticwebapp browse -n MyStaticAppName
+      text: az staticwebapp show -n MyStaticAppName
```

Comment:
I didn't see `az staticwebapp show` was added in this PR and `az staticwebapp browse` was removed in this PR